### ISSUE TITLE
Is the code real

### ISSUE THREAD

**Issue by SummerDream233** [VishnuDuttSharma/deep-multirobot-task#1]
I tried to run it. 
First of all, there are many errors in the code, which looks more like it is not the final version of the code. 
For example, GitHub says that it needs to run coveragetask_ 20rob_ large_ 6FOV_ 8STEP_ 10COMM_ 3layer_ 2filtertap_ GNN2layer_ 32_ 128.json file, but actually the file name in the project is called coveragetask_ 20rob_ FIX_ 6FOV_ 20STEP_ 10COMM_ 3layer_ 2filtertap_ GNN2layer_ 32_ 128.json。 There are many such errors, but they are all small problems. The modified code can still run.
Finally, I generated the data set according to the instructions on GitHub and conducted 1500 training (the training command was strictly in accordance with the instructions on GitHub). The final result shows as follows:
`Train Accuracy: 99.91666666666667%`
`Train Loss: 0.009591424837708473`
`Num agents: 20`
`(40, 20, 5) (40, 20, 5)`
`Valid Accuracy: 33.0%`
`Valid Loss: 18.934585571289062`
`Num agents: 20`
`(40, 20, 5) (40, 20, 5)`
`Test Accuracy: 32.875%`
`Test Loss: 19.36847686767578`
Why? This is completely different from the result of the paper. Isn't it true that the paper has an accuracy rate of about 88%? Why is there only about 33% here?
Seriously, I really hope that this code is true and the final effect is really like that described in the paper. But after I read your source code, I felt deeply worried (your work should be based on the code of Qingbiao Li, and I have carefully read his code)

**Comment by VishnuDuttSharma** [comment]
Thanks for pointing out the issues. The code has been updated to reflect the correct path and options.  We'll also upload the data and pre-trained model before closing the issue. The earlier README was written in an indicative manner. It has also been updated. Apologies for the inconvenience. 

About the accuracy: the paper reports coverage performance, which is different from prediction accuracy. Prediction accuracy checks if the predicted action is exactly the same as the expert action. This metric does not consider the efficacy of a suboptimal action, which may cover fewer rewards than expert action, but still perform better compared to other possible actions.  We have updated README to mention this and moved the code to check reward coverage up. 

The accuracy you obtained suggests overfitting which we did not observe. We're testing this code and will add updates to the thread.

**Comment by VishnuDuttSharma** [comment]
The link to the model and data have been added to the README. 

The code seems to work fine at our end i.e. we were able to train the model without overfitting in 1500 epochs and provide good coverage. The accuracy was around 55% for this model. Please try other checkpoints to see if they provide good performance without overfitting.

**Comment by SummerDream233** [comment]
Thank you for your patient answer and arrangement : )
